packages/@okta/vuepress-site/docs/guides/build-custom-ui-mobile/main/index.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Build a custom sign-in UI in your mobile app
+excerpt: Learn how to build a custom sign-in UI in your mobile app.
+layout: Guides
+---
+
```

Comment:
The current Nutrition facts template says the stack-selector should go above the summary?